package.json

```diff
@@ -41,19 +41,21 @@
     "ajv": "^8.11.2",
     "ajv-formats": "^2.1.1",
     "axios": "^1.3.2",
-    "decimal.js": "^10.4.3",
     "copyfiles": "^2.4.1",
+    "decimal.js": "^10.4.3",
     "ethers": "^5.7.2",
     "exponential-backoff": "^3.1.1",
     "husky": "^8.0.2",
     "jimp": "^0.22.4",
     "js-yaml": "^4.1.0",
+    "langchain": "^0.0.134",
```

Comment:
Do we need this package? It is not used anywhere.

Reply:
I was using it previously and think that refining this command later, I personally, would use langchain but I did see mention on an issue that LangChain is being avoided wasn't sure if that's still the case. I'll remove it for now as it's not being used atm